servers/v3-proxy-api/src/routes/v3Add.ts

```diff
@@ -28,28 +29,33 @@ const v3AddController = async (
   const result = validationResult(req);
   const data = matchedData(req, { includeOptionals: false }) as V3AddParams;
   if (!result.isEmpty()) {
-    // Send validation error to error handling middleware
-    return next(
-      new InputValidationError(result.array({ onlyFirstError: true })[0]),
-    );
-  }
-  try {
-    const variables = buildVariables(data);
-    const headers = req.headers;
-    const accessToken = (data.access_token as string) ?? null;
-    const consumerKey = (data.consumer_key as string) ?? null;
-    const client = getClient(accessToken, consumerKey, headers);
-    const graphResponse = await processV3Add(client, variables, data.tags);
-    return res.json(graphResponse);
-  } catch (err) {
-    // Pass along to error handling middleware
-    // Has to be in a try/catch block due to async call
-    return next(err);
+    throw new InputValidationError(result.array({ onlyFirstError: true })[0]);
   }
+  const variables = buildVariables(data);
+  const headers = req.headers;
+  const accessToken = (data.access_token as string) ?? null;
+  const consumerKey = (data.consumer_key as string) ?? null;
+  const client = getClient(accessToken, consumerKey, headers);
+  const graphResponse = await processV3Add(client, variables, data.tags);
+  return graphResponse;
 };
 
-router.get('/', checkSchema(V3AddSchema, ['query']), v3AddController);
-router.post('/', checkSchema(V3AddSchema, ['body']), v3AddController);
+router.get(
+  '/',
+  checkSchema(V3AddSchema, ['query']),
+  asyncHandler(async (req, res, next) => {
```

Comment:
wrap the method in asyncHandler -- if throws for any reason, will be captured and sent along to error handling middleware